.github/workflows/ci.yml

```diff
@@ -712,8 +712,8 @@ jobs:
           ABFS_ACCESS_KEY:
           S3_BUCKET:
           AWS_REGION:
-          TRINO_AWS_ACCESS_KEY_ID:
-          TRINO_AWS_SECRET_ACCESS_KEY:
+          DATABRICKS_AWS_ACCESS_KEY_ID:
+          DATABRICKS_AWS_SECRET_ACCESS_KEY:
```

Comment:
Could you fix `AbstractSinglenodeDeltaLakeDatabricks` instead?

Reply:
WDYM? Rename the env vars there? The root cause is that both `build-pt` and `pt` jobs need to have the same env vars defined. Product test environments check if an env var is set to produce a more actionable error message. It's much harder to check this when the environment starts, instead of when it's created.

Reply:
> Rename the env vars there? 

Yes. We will replace these environment variables `DATABRICKS_` with `TRINO_` in the future.

Reply:
What about `DATABRICKS_AWS_SESSION_TOKEN`?

Reply:
I renamed them in the env, PTAL.